1.10/administering-clusters/recovering-agent-disk-space.md

```diff
@@ -0,0 +1,12 @@
+---
+post_title: Recovering Agent Disk Space
+menu_order: 900
+---
+
+If tasks fill up the reserved volume of an agent node, there are a few options to recover space:
+
+- If the work directory is on a separate volume (as recommended in [Agent nodes](/docs/1.10/installing/custom/system-requirements/#agent-nodes), then you can empty this volume and restart the node.
```

Comment:
Careful about using `agent` vs `node`.  In this case, the minimum required is to stop the Mesos agent (e.g. `systemctl stop dcos-mesos-slave`), clear the volume, and then start the Mesos agent.  No need to restart the entire node.